src/Server/OutputBuilder.cs

```diff
@@ -23,13 +24,37 @@ public class OutputBuilder
 
         private int bufferPos;
         private int charsCapacity;
+        
+        //Static Char Numbers
+        private static readonly char[] CharNumbers = {'0', '1', '2', '3', '4', '5', '6', '7', '8', '9'};
+
+        private static readonly string[] CharBool = {"False", "True"};
 
         /// <summary>
         /// Temporary string used for the replace method.
         /// </summary>
         private List<char> replacement;
+        
+        /// <summary>
+        /// Quickly gets buffer length of int
+        /// </summary>
+        /// <param name="n"></param>
+        /// <returns>buffer length</returns>
+        private static int GetIntLength(long n)
```

Comment:
`int n` instead of long?  Or `GetLongLength` and adjust to account for long's bigger range, and fix naming/comments?